src/current/v23.1/sso-db-console.md

```diff
@@ -6,94 +6,104 @@ docs_area: manage
+
+## Configure your cluster to use an OIDC client and provider
+
+**Prerequisites**:
 
-	<img src="{{ 'images/v23.1/google-oidc-client.png' | relative_url }}" alt="Google OAuth 2.0 client details" style="border:1px solid #eee;max-width:100%" />
+You must have the ability to update your cluster settings, which requires a SQL user either the [admin role]({% link {{ page.version.version }}/security-reference/authorization.md %}#admin-role) or the `MODIFYCLUSTERSETTING` role option.
 
-1. Add the callback URL to the list of **Authorized redirect URIs**. On a local cluster, this will be `https://localhost:8080/oidc/v1/callback`. You will later set `server.oidc_authentication.redirect_url` to the same value.
+### Cluster Settings
 
-1. Open a SQL shell to the cluster on node 1:
+The following [cluster settings]({% link {{ page.version.version }}/cluster-settings.md %}) must be configured to enable SSO authentication to the DB Console.
+
+Use the [Set Cluster Setting Statement]({% link {{ page.version.version }}/set-cluster-setting.md %}) to configure your cluster settings.
+
+| Cluster Setting | Description 
+|-----------------|------
+| `server.oidc_authentication.enabled` | A Boolean that enables or disables SSO
+| `server.oidc_authentication.client_id` | Your auth client's ID, e.g., `32789079457-g3hdfw8cbw85obi5cb525hsceaqf69unn.apps.googleusercontent.com`
+| `server.oidc_authentication.client_secret` | Your auth client's secret
+| `server.oidc_authentication.redirect_url` | specifies the callback URL that redirects the user to CockroachDB after a successful authentication. This can be the address of a node in the cluster or the address of a load balancer that routes traffic to the nodes. The path must be appended with `/oidc/v1/callback`. E.g., `https://{ your cluster's domain } :8080/oidc/v1/callback`.
+| `server.oidc_authentication.provider_url` | Specifies the OAuth issuer identifier. Ensure that the URL does not have a terminating `/`. For more information, see the [OIDC specification](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfig). Note that CockroachDB appends the required `/.well-known/openid-configuration` by default. You do not need to include it. E.g., `https://accounts.google.com`
+| `server.oidc_authentication.scopes` | A space-delimited list of the [OAuth scopes](https://openid.net/specs/openid-connect-core-1_0.html#ScopeClaims) being requested for an Access Token. The `openid` and `email` scopes must be included. E.g., `openid profile email`.
+| `server.oidc_authentication.claim_json_key` | The field/key used to identify the user from the external identity provider's [ID Token](https://openid.net/specs/openid-connect-core-1_0.html#IDToken). Must be set to `email`.
```

Comment:
It is incorrect that this must be set to email. Any field can be used that is present in the token.